global.json

```diff
@@ -1,3 +1,3 @@
 {
-    "sources": ["src"]
+    "sources": ["src", "test"]
```

Comment:
what exactly pushed this?  we don't need similar reference to top-level test directory in MVC folder.